luigi_pipeline/tests/test_hail_tasks.py

```diff
@@ -17,30 +18,69 @@ def tearDown(self):
         # Remove the directory after the test
         shutil.rmtree(self.test_dir)
 
-    def test_hail_matrix_table_load(self):
-        temp_dest_path = os.path.join(self.test_dir, 'temp_test_load.mt')
-        task = HailMatrixTableTask(source_paths=[TEST_DATA_MT_1KG], dest_path=temp_dest_path)
+    def _temp_dest_path(self):
+        return os.path.join(self.test_dir, 'temp_test_load.mt')
+
+    def _hail_matrix_table_task(self):
+        temp_dest_path = self._temp_dest_path()
+        return HailMatrixTableTask(source_paths=[TEST_DATA_MT_1KG],
+                                   dest_path=temp_dest_path,
+                                   genome_version='37')
+
+    def _set_validation_configs(self):
+        global_config = GlobalConfig()
+        global_config.param_kwargs['validation_37_coding_ht'] = global_config.validation_37_coding_ht = \
+            'tests/data/validation_37_coding.ht'
+        global_config.param_kwargs['validation_37_noncoding_ht'] = global_config.validation_37_noncoding_ht = \
+            'tests/data/validation_37_noncoding.ht'
```

Comment:
Is there a Luigi configuration file for tests that these could be read from?

Reply:
Yeah I briefly looked into specifying a config file in code, which would be a nice to have for generic changes, but couldn't find it in the allocated time :(

Reply:
Ok. It's a little unfortunate that tests have to mock both the global_config attribute and a value in global_config.param_kwargs.